app/pb_kits/playbook/pb_typeahead/typeahead.rb

```diff
@@ -5,12 +5,19 @@ module PbTypeahead
     class Typeahead
       include Playbook::Props
 
+      prop :async, type: Playbook::Props::Boolean,
+                    default: false
```

Comment:
I don't think this props really is necessary due to the fact that it should be implied that `load_options` means "async" inherently.